README.md

```diff
@@ -1375,6 +1375,15 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='no_direct_standard_out'></a>(<a href='#no_direct_standard_out'>link</a>) **Prefer dedicated logging systems like [`os_log`](https://developer.apple.com/documentation/os/logging) or [`swift-log`](https://github.com/apple/swift-log) over writing directly to standard out using `print(…)`, `logDebug(…)`, or `dump(…)`.**
```

Comment:
Should we use hyphens between words in the slug like we do for other rules?

Reply:
Fixed in https://github.com/airbnb/swift/pull/119/commits/4a8d457185e58819bc34a2f6040cb488df7c4d22